### PR TITLE
[DSD-5387] added certify properties for different plugins

### PR DESCRIPTION
**Pull request by challabeehyv**

### CODE REVIEW

File: certify-insurance.properties [hunk 1]
```diff
@@ -224,3 +224,145 @@ mosip.certify.key-values={\
...
+mosip.certify.vciplugin.sunbird-rc.credential-type.VehicleInsuranceCredential.registry-search-url=http://10.3.148.107/registry/api/v1/Insurance/search
+
+##---------------------------------Insurance Credential Configurations------------------------------------------------------
+
+mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.static-value-map.issuerId=did:web:holashchand.github.io:test_project:32b08ca7-9979-4f42-aacc-1d73f3ac5322
+mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.template-url=${spring_config_url_env}/*/${active_profile_env}/${spring_config_label_env}/insurance-credential.json
+mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.registry-get-url=http://10.3.148.107/registry/api/v1/Insurance/
+mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.cred-schema-id=did:schema:0d10a2cf-94de-4ffc-b32c-4f1a61ee05ba
+mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.cred-schema-version=1.0.0
+mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.registry-search-url=http://10.3.148.107/registry/api/v1/Insurance/search
```
Comment (vishwa-vyom) [inline certify-insurance.properties:368]: Plugin properties are suppose to go to the certify-plugin-insurance.properties

Comment (vishwa-vyom) [reply certify-insurance.properties:368]: same way for the mock also

File: certify-mock.properties [hunk 2]
```diff
@@ -37,190 +37,201 @@ mosip.certify.authn.allowed-audiences={ '${mosip.certify.domain.url}${server.ser
...
-                            'dob': {'display': {{'name': 'Date of Birth','locale': 'en'}}},\
-                            'gender': {'display': {{'name': 'Gender','locale': 'en'}}},\
-                            'benefits': {'display': {{'name': 'Benefits','locale': 'en'}}},\
-                            'email': {'display': {{'name': 'Email Id','locale': 'en'}}},\
-                            'policyIssuedOn': {'display': {{'name': 'Policy Issued On','locale': 'en'}}},\
-                            'policyExpiresOn': {'display': {{'name': 'Policy Expires On','locale': 'en'}}},\
-                            'policyName': {'display': {{'name': 'Policy Name','locale': 'en'}}},\
-                            'policyNumber': {'display': {{'name': 'Policy Number','locale': 'en'}}}\
+                            'name': { 'display': {{'name': 'Given Name', 'locale': 'en' }}}, \
+                             'age': { 'display': {{ 'name': 'Age', 'locale': 'en'}}}\
```
Comment (vishwa-vyom) [inline certify-mock.properties:82]: why only name and age ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: certify-insurance.properties (hunk 1 above)
Comment (challabeehyv) [reply certify-insurance.properties]: added separate files for plugin properties

File: certify-mock.properties (hunk 2 above)
Comment (challabeehyv) [reply certify-mock.properties]: added extra properties

**Pull request closed without merging**